docs/appendices/release-notes/5.4.4.rst

```diff
@@ -46,6 +46,9 @@ See the :ref:`version_5.4.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed an issue that led to file not found errors when trying to restore a
+  snapshot that was taken after a table had been swapped.
```

Comment:
This description reads to me that we fixed it for existing snapshots as well, so fixed during restore.
But isn't the fix only working when doing a new snapshot? If so, shouldn't we mentioned this?
```suggestion
- Fixed an issue that led to file not found errors when trying to restore a
  snapshot that was taken after a table had been swapped. A new snapshot 
  must be taken to apply the fix and solve the issue.
```

Reply:
Sounds good. Can you add the change and merge this?

Reply:
Sure will do.